lib/propagate-steps.unit.spec.js

```diff
@@ -62,9 +62,7 @@ test('When adding _parent properties to step instances', function (t) {
 test('When a step is referenced by multiple parent pages', function (t) {
   const badData = getTestData('input-duplicate-step-ref')
   try {
-    /* eslint-disable no-unused-vars */
-    const instances = propagate(badData)
-    /* eslint-enable no-unused-vars */
+    propagate(badData)
```

Comment:
I'd add a line like so after the `propagate` as the test above has
```
t.notOk('This test should have thrown an error')
```
so that the test can't silently pass

Reply:
Fixed with t.throws